ocfweb/main/templates/main/home.html

```diff
@@ -7,11 +7,24 @@
 {% block hero %}
 <div class="ocf-penguin-swing"></div>
 
-<div class="ocf-hero">
+<div class="ocf-hero" style="background-color:#888">
```

Comment:
black text with gray background is hard to read. I would change the background color of the hero element to be lighter. You can't invert the colors since then the penguin will disappear